ipamutils/utils.go

```diff
@@ -13,11 +14,24 @@ var (
 	// PredefinedGranularNetworks contains a list of 64K IPv4 private networks with host size 8
 	// (10.x.x.x/24) which do not overlap with the networks in `PredefinedBroadNetworks`
 	PredefinedGranularNetworks []*net.IPNet
-
-	initNetworksOnce sync.Once
+	// ErrPoolsAlreadyInitialized notifies the default pools are already set
+	ErrPoolsAlreadyInitialized = fmt.Errorf("predefined address pools are already initialized")
```

Comment:
I think you don't need to define the error this way in the top especially since it's used in just one place below it seems. Good enough to return `fmt.Errorf` directly from the point in the code where you raise the error below.